core/remote/cozy.js

```diff
@@ -127,6 +127,10 @@ class RemoteCozy {
     return this.client.auth.unregisterClient()
   }
 
+  update() /*: Promise<void> */ {
+    return this.client.auth.updateClient()
```

Comment:
I'm surprised to not see some params here: `this.client.auth.updateClient(params)`, coming from https://github.com/cozy-labs/cozy-desktop/blob/master/core/remote/registration.js#L61-L88

Reply:
The params are set automatically in `cozy-client-js` https://github.com/cozy/cozy-client-js/blob/master/src/index.js#L220.

This is the same thing for `unregisterClient()`.